ThunderCloud/AccordionTabBarViewController.swift

```diff
@@ -101,7 +101,7 @@ public class AccordionTabBarItem: Row {
         accordionCell.cellTextLabel?.textColor = contrastingColor
         accordionCell.cellImageView?.tintColor = contrastingColor
         
-        accordionCell.topConstraint.constant = isFirstItem ? 28 : 8
+        accordionCell.topConstraint.constant = isFirstItem ? UIApplication.shared.statusBarFrame.height : 0
```

Comment:
Not sure why `28` was used here and `20` above, do we need `8` padding?

Reply:
Previously this was the spacing between the top of the cell and the image/text (Kinda like padding) now it's more like margin, and the 8 points of vertical padding isn't really needed any more as we rely on aligning centre vertical with the parent (And also > constants of 0)